### ISSUE TITLE
ImportError: cannot import name 'exec_command' from 'PyInstaller.utils.hooks'

### ISSUE THREAD

**Issue by tobinjones** [pyinstaller/pyinstaller#3689]
Building a PySide2/QtQuick application with the development branch of pyinstaller gives the following error:

 ```
 File "<snip>\PyInstaller\hooks\hook-PySide2.QtQuick.py", line 13, in <module>
    from PyInstaller.utils.hooks import get_qmake_path, exec_command
ImportError: cannot import name 'exec_command' from 'PyInstaller.utils.hooks' (<snip>\PyInstaller\utils\hooks\__init__.py)
```
(note: using Python3.7, Windows 10)

I suspect that `hook-PySide2.QtQuick.py` should instead be importing `exec_command_stdout`. Possibly related to issue #3089

**Comment by bjones1** [comment]
Needs fixing! Please provide a PR.

**Comment by patrickelectric** [comment]
any update with this issue ?

**Comment by Vyeadiai** [comment]
I am experiencing this issue with PySide2.QtQuick.

`4733 INFO: Loading module hook "hook-PySide2.QtQuick.py"...
Traceback (most recent call last):
  File "/Users/ed/.pyenv/versions/3.5.7/bin/pyinstaller", line 11, in <module>
    sys.exit(run())
  File "/Users/ed/.pyenv/versions/3.5.7/lib/python3.5/site-packages/PyInstaller/__main__.py", line 111, in run
    run_build(pyi_config, spec_file, **vars(args))
  File "/Users/ed/.pyenv/versions/3.5.7/lib/python3.5/site-packages/PyInstaller/__main__.py", line 63, in run_build
    PyInstaller.building.build_main.main(pyi_config, spec_file, **kwargs)
  File "/Users/ed/.pyenv/versions/3.5.7/lib/python3.5/site-packages/PyInstaller/building/build_main.py", line 838, in main
    build(specfile, kw.get('distpath'), kw.get('workpath'), kw.get('clean_build'))
  File "/Users/ed/.pyenv/versions/3.5.7/lib/python3.5/site-packages/PyInstaller/building/build_main.py", line 784, in build
    exec(text, spec_namespace)
  File "<string>", line 17, in <module>
  File "/Users/ed/.pyenv/versions/3.5.7/lib/python3.5/site-packages/PyInstaller/building/build_main.py", line 241, in __init__
    self.__postinit__()
  File "/Users/ed/.pyenv/versions/3.5.7/lib/python3.5/site-packages/PyInstaller/building/datastruct.py", line 158, in __postinit__
    self.assemble()
  File "/Users/ed/.pyenv/versions/3.5.7/lib/python3.5/site-packages/PyInstaller/building/build_main.py", line 500, in assemble
    module_hook.post_graph()
  File "/Users/ed/.pyenv/versions/3.5.7/lib/python3.5/site-packages/PyInstaller/building/imphook.py", line 410, in post_graph
    self._load_hook_module()
  File "/Users/ed/.pyenv/versions/3.5.7/lib/python3.5/site-packages/PyInstaller/building/imphook.py", line 377, in _load_hook_module
    self.hook_module_name, self.hook_filename)
  File "/Users/ed/.pyenv/versions/3.5.7/lib/python3.5/site-packages/PyInstaller/compat.py", line 736, in importlib_load_source
    return mod_loader.load_module()
  File "<frozen importlib._bootstrap_external>", line 418, in _check_name_wrapper
  File "<frozen importlib._bootstrap_external>", line 841, in load_module
  File "<frozen importlib._bootstrap_external>", line 700, in load_module
  File "<frozen importlib._bootstrap>", line 268, in _load_module_shim
  File "<frozen importlib._bootstrap>", line 693, in _load
  File "<frozen importlib._bootstrap>", line 673, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 697, in exec_module
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File "/Users/ed/.pyenv/versions/3.5.7/lib/python3.5/site-packages/PyInstaller/hooks/hook-PySide2.QtQuick.py", line 13, in <module>
    from PyInstaller.utils.hooks import get_qmake_path, exec_command
ImportError: cannot import name 'exec_command'` 


Any resolution on this? This has been blocking me for days with no end in sight. I have tried with multiple versions of Python. Thanks

**Comment by adynathos** [comment]
The pull request #4206 solves the problem, I am able to package a PySide2 program successfully. Thank you very much for the fix.

Until this gets merged, you can install the fixed version from this branch: https://github.com/sjackso/pyinstaller/tree/fix_3689

**Comment by bjones1** [comment]
This was fixed in 5b5791285332003940f4bc70c32bb0cf10a32917. Please reopen if there are still problems.
